### ISSUE TITLE
Support Request: redirect URL does not match

### ISSUE THREAD

**Issue by SkyroOryks** [d0n4v4nb3ck3r/homebridge-honeywell-home#739]
### Describe Your Problem

trying to get everything registered with the honeywell developer site, I made a new app, inserted the callback url, and put in the key and secret codes, but i keep getting a 400 error. I made sure to type the callback exactly as shown in instructions. even tried forwarding the port.

### Relevant log output

```shell
{"code":400, "message":"The redirect URL provided does not match the redirect URL registered for the app."}
```

### Screenshots

![bug issue](https://user-images.githubusercontent.com/13127174/187053836-6a8ce9b4-de7e-4322-afde-163c1fe9d4d5.png)

### Device & Model

Raspberry Pi Zero W

### Node.js Version

v16.16.0

### NPM Version

dont know how to find

### Homebridge Version

v1.5.0

### Homebridge Honeywell Home Plugin Version

v11.4.2

### Homebridge Config UI X Plugin Version

v4.50.1

### Operating System

Raspbian GNU/Linux Bullseye (11)

**Comment by donavanbecker** [comment]
@SkyroOryks The log tells you what is wrong. You have to use the same redirect url as you have In the Honeywell developer app.

**Comment by SkyroOryks** [comment]
Yes, I copied exactly what the system told me to copy for the callback url. I dont know why it’s coming up different somehow. http://homebridge.local:8585/

**Comment by saimbhi** [comment]
Its because there is a bug or changes on the honeywell side. 
Following change takes you further if you do some tweeks on the servers.js but then it never really completes:
https://api.honeywell.com/oauth2/authorize?response_type=code&redirect_uri=http://MYLOCALIP:8585/&client_id=XXXXXXXXXXXXXXXXX

Following doesn't work which is called when you click on "link account"
https://api.honeywell.com/oauth2/authorize?response_type=code&redirect_uri=http://MYLOCALIP:8585/auth&client_id=XXXXXXXXXXXXXXXXX

**Comment by patricksmith0330** [comment]
@donavanbecker I have having the same issue. This is definitely a bug.

**Comment by clickbg** [comment]
Same here, I tried multiple URLs in https://developer.honeywellhome.com/ but none worked. This was an installation that was working fine until Resideo took over.

**Correction**: You have to change the URL in https://developer.honeywellhome.com/ to http://YOUR_URL:8585/auth The change is adding /auth to the URL. That worked for me, you can get the correct app URL by coping the address from the error popup and looking at the redirect_uri= part.

**Comment by donavanbecker** [comment]
This should be corrected in v11.4.3
